x-pack/plugins/security_solution/public/entity_analytics/pages/entity_store_management_page.tsx

```diff
@@ -0,0 +1,456 @@
+              showClearModal();
+            }}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.clear"
+              defaultMessage="Clear"
+            />
+          </EuiButton>
+        </EuiPanel>
+        {isClearModalVisible && (
+          <EuiConfirmModal
+            isLoading={deleteEntityEngineMutation.isLoading}
+            title={
+              <FormattedMessage
+                id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.clearEntitiesModal.title"
+                defaultMessage="Clear Entity data?"
+              />
+            }
+            onCancel={closeClearModal}
+            onConfirm={() => {
+              deleteEntityEngineMutation.mutate();
+            }}
+            cancelButtonText={
+              <FormattedMessage
+                id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.clearEntitiesModal.close"
+                defaultMessage="Close"
+              />
+            }
+            confirmButtonText={
+              <FormattedMessage
+                id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.clearEntitiesModal.clearAllEntities"
+                defaultMessage="Clear All Entities"
+              />
+            }
+            buttonColor="danger"
+            defaultFocusedButton="confirm"
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.clearConfirmation"
+              defaultMessage={
+                'This will delete all Security Entity store records. Source data, Entity risk scores, and Asset criticality assignments are unaffected by this action. This operation cannot be undone.'
+              }
+            />
+          </EuiConfirmModal>
+        )}
+      </>
+    );
+  };
+
+  const FileUploadSection: React.FC = () => {
+    if (
+      !hasEntityAnalyticsCapability ||
+      !isAssetCriticalityEnabled ||
+      assetCriticalityPrivilegesError?.body.status_code === 403
+    ) {
+      return <AssetCriticalityIssueCallout />;
+    }
+    if (!hasAssetCriticalityWritePermissions) {
+      return <InsufficientAssetCriticalityPrivilegesCallout />;
+    }
+    return (
+      <EuiFlexItem grow={3}>
+        <EuiTitle size="s">
+          <h2>
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.assetCriticalityUploadPage.subTitle"
+              defaultMessage="Upload entities using CSV file upload"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="m" />
+        <EuiText size="s">
+          <FormattedMessage
+            id="xpack.securitySolution.entityAnalytics.assetCriticalityUploadPage.description"
+            defaultMessage="Bulk assign asset criticality by importing a CSV, TXT, or TSV file exported from your asset management tools. This ensures data accuracy and reduces manual input errors."
```

Comment:
I'm a little confused by this UI. It looks exactly like the asset criticality upload page but with a different title.
Do both of these pages (Entity store and Asset criticality) do the same thing? As a user, I would be confused as to what the difference is between the two pages and when I should use one or the other.

It's also confusing that the level 2 heading of this page talks about uploading entities, but the description and the "What is" text talk about asset criticality (for which we have a separate page).




![image](https://github.com/user-attachments/assets/5c8225e9-d675-4386-a56f-ef3b7211ebf5)
![image](https://github.com/user-attachments/assets/36d5871c-c1ea-45ac-91ca-2fc1c9619be7)

Reply:
@jaredburgettelastic and @joedatlive are probably better suited to answer these questions.

From my understanding, the difference is that we can enable the entity store in this page. We also want to provide an option for users to upload entities, which right now is done via asset criticality bulk upload. That is why these two pages are so similar, apart from the Entity store toggle.

Reply:
Oh I didn't even see the toggle...On the risk scoring page the toggle is under the Status heading and a bit more noticeable.
I guess even though the underlying functionality for uploading entities is the same as for uploading asset criticality, from a user's perspective I would expect the description and the help text on this page to focus on the entity store / uploading entities rather than on asset criticality.

Reply:
This makes sense, @joedatlive and @r4zr32d3k1l let's sync up on this to determine if there's anything else we want to do to make the user expectations more clear